superset-frontend/src/dashboard/components/menu/DownloadMenuItems/DownloadAsPdf.tsx

```diff
@@ -29,7 +29,7 @@ export default function DownloadAsPdf({
 
   return (
     <Menu.Item key="download-pdf" {...rest}>
-      <div onClick={onDownloadPdf} role="button" tabIndex={0}>
+      <div role="button" onClick={onDownloadPdf} tabIndex={0}>
```

Comment:
Wondering if we can just bring the onClick and tabIndex to the parent (Menu.Item) and get rid of that child `<div>` altogether here